ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -897,6 +902,30 @@ func checkK8sJobAndUpdateStatusIfNeeded(ctx context.Context, rayJob *rayv1.RayJo
 			return true
 		}
 	}
+	if rayv1.IsJobTerminal(rayJob.Status.JobStatus) && rayJob.Status.JobDeploymentStatus == rayv1.JobDeploymentStatusRunning {
+		submitterGracePeriodTime, err := strconv.Atoi(os.Getenv(utils.SUBMITTER_GRACE_PERIOD_TIME))
+		if err != nil {
+			logger.Info(
```

Comment:
This is too verbose. Please remove it. Currently, KubeRay logs are very verbose, and we are trying to improve it.